modules/administration/pages/ansible-setup-control-node.adoc

```diff
@@ -140,4 +140,4 @@ ansible client240.mgr.example.org -m ping
 ----
 
 You may now run remediations
```

Comment:
Drive-by edit:
```suggestion
You may now run remediations.
```
BTW, my spell checker does not like the plural of remediation.